rotkehlchen/tests/utils/api.py

```diff
@@ -152,6 +180,9 @@ def wait_for_async_task(
                 )
             status = json_data['result']['status']
             if status == 'completed':
+                # Move status code to the outcome dict for easier checking
+                status_code = json_data['result'].get('status_code')
```

Comment:
nitpick: I would have avoided the get sintax since this defaults to None and instead would have used the dict access operator (`['status_code']`). Otherwise you are adding the key even if the json data didn't have it

Reply:
Yeah and this is intended since successful responses don't return a status code. For example this https://github.com/rotki/rotki/blob/e44c8654fb3e3bd4d8b890f0d03a1b952af87229/rotkehlchen/api/rest.py#L1972 or `OK_RESULT` constant or any other asynchronous successful response. Does it make sense?

Reply:
so if there is not status code in the async response the result of this function will have `None` in there?

Reply:
Yes. Is it bad?

Reply:
no I don't think so. At least can't think of a reason off the top of my head. Just asked to have it clear in my mind how it works.